Makefile

```diff
@@ -94,9 +96,29 @@ deploy: manifests
 
 # Generate manifests e.g. CRD, RBAC etc.
 manifests:
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all
-	kustomize build config  -o deploy/gatekeeper.yaml
-	bash -c 'for x in vendor/github.com/open-policy-agent/frameworks/constraint/deploy/*.yaml ; do echo --- >> deploy/gatekeeper.yaml ; cat $${x} >> deploy/gatekeeper.yaml ; done'
+	@go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all
+	@kustomize build config  -o deploy/gatekeeper.yaml
+	@kustomize build config -o ${CHART_PATH}/helm-modifications/_temp.yaml
+	@bash -c 'for x in vendor/github.com/open-policy-agent/frameworks/constraint/deploy/*.yaml ; do echo --- >> deploy/gatekeeper.yaml ; cat $${x} >> deploy/gatekeeper.yaml ; done'
+	@bash -c 'for x in vendor/github.com/open-policy-agent/frameworks/constraint/deploy/*.yaml ; do echo --- >> ${CHART_PATH}/helm-modifications/_temp.yaml ; cat $${x} >> ${CHART_PATH}/helm-modifications/_temp.yaml ; done'
+	@kustomize build ${CHART_PATH}/helm-modifications -o ${CHART_PATH}/templates/gatekeeper.yaml
+	@sed -i -E "s/STATEFUL_SET_RESOURCES/\
```

Comment:
I'm not in love with the `sed` thing. That being said, I'm not sure I can come up with a better solution that doesn't add more dependencies to the build server.

We should:

   * Add a likely-to-be-unique prefix to these anchor values to help avoid collision, ex.: HELMSUBST_STATEFUL_SET_RESSOURCES
  * create a bash script that lives in the `helm-modifications` dir so that all helm-related stuff is located close to each other and to avoid cluttering the makefile